### PR TITLE
support generate normal public class for kotlin and scala. 

### PR DESCRIPTION
**Pull request by jxnu-liguobin**
https://github.com/kobylynskyi/graphql-java-codegen/issues/524.
https://github.com/kobylynskyi/graphql-java-codegen/issues/548

1.Support generate normal public class for kotlin and scala. (Not support java 14 record.)
2.Fix response generation, return type not generated correctly (missing `?` in kotlin).

### CODE REVIEW

**Comment by ntf** [comment]
I quickly scan through the change set for Kotlin and notice that you also generate `equals` and `hashCode` which is great. 

LGTM for the kotlin change for #524

File: src/main/resources/templates/kotlin-lang/kotlinClassGraphqlType.ftl [hunk 1]
```diff
@@ -11,6 +11,11 @@ import ${import}.*
 <#if toStringForRequest>
 import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLRequestSerializer
 </#if>
+<#if generateModelOpenClasses>
```
Comment (kobylynskyi) [inline src/main/resources/templates/kotlin-lang/kotlinClassGraphqlType.ftl:14]: @jxnu-liguobin please merge nested if statement:
```ftl
<#if generateModelOpenClasses && equalsAndHashCode>
```

(relevant to all .ftl classes)

**Review by kobylynskyi — CHANGES_REQUESTED** [review]

**Review by kobylynskyi — APPROVED** [review]
@jxnu-liguobin you are fast 🚀 
Merging it now.

**Comment by jxnu-liguobin** [comment]
> @jxnu-liguobin you are fast 🚀
> Merging it now.

😸 We are in the daytime now.

**Pull request merged**